examples/bart_vs_bikes.py

```diff
@@ -31,10 +31,11 @@
 )
 
 
-@st.cache(persist=True)
+@st.cache
 def from_data_file(filename):
-    dirname = os.path.dirname(__file__)
-    return pd.read_json(os.path.join(dirname, "data", filename))
+    dirname = "https://raw.githubusercontent.com/streamlit/streamlit/develop/examples/"
+    url = os.path.join(dirname, "data", filename)
+    return pd.read_json(url)
```

Comment:
I was expecting this example to be deleted as you moved it into `hello.py`. And at that point we'd move the data outside this repo too.

But if you want to merge this PR as a waypoint to that final state, that's fine.

Reply:
+1